src/Umbraco.Core/Services/PropertyValidationService.cs

```diff
@@ -102,6 +111,17 @@ public IEnumerable<ValidationResult> ValidatePropertyValue(
         }
     }
 
+    [Obsolete("Please use the overload that accepts a PropertyValidationContext. Will be removed in V16.")]
+    public IEnumerable<ValidationResult> ValidatePropertyValue(
+        IDataEditor editor,
+        IDataType dataType,
+        object? postedValue,
+        bool isRequired,
+        string? validationRegExp,
+        string? isRequiredMessage,
+        string? validationRegExpMessage)
+        => ValidatePropertyValue(editor, dataType, postedValue, isRequired, validationRegExp, isRequiredMessage, validationRegExpMessage, PropertyValidationContext.Empty());
+
     /// <inheritdoc />
     public bool IsPropertyDataValid(IContent content, out IProperty[] invalidProperties, CultureImpact? impact)
```

Comment:
This entire `.Where` smells a bit like it could use an extraction into a method or something, instead of a giant `.Where()`
Might be out of scope for now 🙈